### PR TITLE
cpdp-525: BPR now retreived for individuals with SA UTRS

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala [hunk 1]
```diff
@@ -59,8 +63,8 @@ class CGTPropertyDisposalsConnectorImpl @Inject()(http: HttpClient, servicesConf
   ): EitherT[Future, Error, HttpResponse] =
    EitherT[Future, Error, HttpResponse](
     entity.fold(
-      t => http.get(s"$bprBaseUrl/sautr/${t.sautr.value}"),
-      i => http.post(s"$bprBaseUrl/nino/${i.nino.value}", Json.toJson(BprRequest(i.name.firstName, i.name.lastName, i.dateOfBirth.value)))
+      t => http.post(bprUrl(Left(t.sautr)), JsNull),
+      i => http.post(bprUrl(i.id), Json.toJson(BprRequest(i.name.firstName, i.name.lastName, i.dateOfBirth.map(_.value))))
```
Comment (co-category) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala]: could these identifiers be made sufficiently descriptive to reflect their semantics

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala]: what kind of thing were you thinking?

Comment (co-category) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala]: t -> trust
i -> individual

Comment (co-category) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala]: generally not a fan of single character identifiers

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala]: oh right, yeah sure

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala [hunk 2]
```diff
@@ -21,16 +21,19 @@ sealed trait UserType
...
                                email: Option[Email]
                              ) extends UserType
 
   final case class Trust(sautr: SAUTR) extends UserType
 
   final case object OrganisationUnregisteredTrust extends UserType
 
-  final case class InsufficientConfidenceLevel(nino: Option[NINO]) extends UserType
+  final case class InsufficientConfidenceLevel(nino: Option[NINO],
+                                               name: Name,
```
Comment (co-category) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala]: Could we rename "InsufficientConfidenceLevel" to "UserWithInsufficientConfidenceLevel" or "EntityWithInsufficientConfidenceLevel"?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/UserType.scala]: I renamed it to "IndividualWithInsufficientConfidenceLevel" - confidence level only really applies to individuals

**Pull request merged**